wolf/CMakeLists.txt

```diff
@@ -154,15 +154,15 @@ elseif (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
   set(CMAKE_CXX_FLAGS_DEBUG "-g")
   set(CMAKE_CXX_FLAGS_RELEASE "-O3")
 elseif (CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
-  # using Microsoft Visual C++
+  # using Visual Studio C++
   # set C++23 as the primary standard
   set(CMAKE_CXX_STANDARD 23)
-  set(CMAKE_CXX_FLAGS "/EHsc /W3 /bigobj")
+  set(CMAKE_CXX_FLAGS "/MDd /EHsc /W4 /bigobj")
```

Comment:
Remove '/MDd' flag because it defines '_DEBUG' and you add this flag later in this file too.

Reply:
It's more than just defining `_DEBUG`. without this there would be link errors about missing `crt` related symbols.

https://learn.microsoft.com/en-us/cpp/build/reference/md-mt-ld-use-run-time-library?view=msvc-170

Reply:
For `crt` add `/MD`.
For `LIBCMT.lib` linking error add `/MT`.

Reply:
okay.